### ISSUE TITLE
Consider our CoT dataset (CoT Collection)

### ISSUE THREAD

**Issue by SeungoneKim** [yaodongC/awesome-instruction-dataset#14]
Hello, thanks for providing this awesome repository introducing different instruction datasets!
Could you consider adding our CoT Collection dataset? It's a massive instruction dataset consisted of 1.84 million rationales across 1060 NLP tasks!

https://arxiv.org/abs/2305.14045

Thank you in advance!

**Comment by sworddish** [comment]
As long as you refactor and release the codes related to the amazing work 👍🏻 @SeungoneKim

**Comment by SeungoneKim** [comment]
Sounds fair enough!! I'll organize the code & upload the models until this week. Thanks in advance!

**Comment by yaodongC** [comment]
Absolutely, if you could create a pull request, I'd be more than happy to review and merge it. Thanks.
